client/src/components/common/Loader/Loader.tsx

```diff
@@ -9,7 +9,7 @@ type LoaderMods = {
 };
 
 type Props = {
-  children: ReactElement | ReactElement[];
+  children?: ReactNode;
   isLoading: boolean;
   showContent?: boolean;
 };
```

Comment:
This change was made to make possible use Loader without children, for initial page loading.